tcms/testruns/tests/test_admin.py

```diff
@@ -91,3 +92,48 @@ def test_changelist_view_icon_column(self):
             """
             ),
         )
+
+    @parameterized.expand(
+        [
+            ("positive", TestExecutionStatus.objects.all().filter(weight__gt=0)),
+            ("neural", TestExecutionStatus.objects.all().filter(weight=0)),
+            ("negative", TestExecutionStatus.objects.all().filter(weight__lt=0)),
+        ]
+    )
+    def test_test_execution_statuses_can_be_deleted_but_one_must_remain(
+        self, name, exe_statuses
```

Comment:
Pylint isn't happy about `name` because it is not in use. Try `_name` instead. If that doesn't work silence this pylint warning.